app/src/main/java/org/mozilla/tv/firefox/session/SessionRepo.kt

```diff
@@ -116,7 +116,7 @@ class SessionRepo(
         webViewCache.doNotPersist()
     }
 
-    private fun exitFullScreenIfPossible() {
+    fun exitFullScreenIfPossible() {
```

Comment:
Is there a way to check if the user is in fullscreen when we try to exit? Might be worth throwing a telemetry event in here, to see how many times the overlay is opened that unfullscreens a video, and we could compare this to total overlay openings/menu press.

Reply:
Upon closer inspection, it looks like there is.  I'm going to update the code to include that check instead of just spamming 'exit fullscreen' calls.

Re: the telemetry ping, I'm not sure I understand what question this would be answering.  Could you clarify?

Reply:
re: telemetry event, add an event that lets us know the actual impact on users - e.g., send an event if user opens overlay AND it unfullscreens. That way we have at least some idea of how many people get forcibly un-fullscreened, which may not be the best experience. Then we can decide, if that number is high, whether to try to find another solution, or if it's a negligible number, let it be.